Rajitha/canArrayNest.js

```diff
@@ -0,0 +1,56 @@
+/**************************************************************************
+* Create a function that returns true if the first array can be nested
+* inside the second.
+* arr1 can be nested inside arr2 if:
+* 1. arr1's min is greater than arr2's min.
+* 2. arr1's max is less than arr2's max.
+* Examples
+* canNest([1, 2, 3, 4], [0, 6]) ➞ true
+* canNest([3, 1], [4, 0]) ➞ true
+* canNest([9, 9, 8], [8, 9]) ➞ false
+* canNest([1, 2, 3, 4], [2, 3]) ➞ false
+****************************************************************************/
+const canArrayNest = (arr1, arr2) => {
+  let tArr1  = [...arr1];
+  let tArr2  = [...arr2];;  
```

Comment:
hmm why are you creating a copy? Are you modifying the existing arrays? If not, there is no reason to create a copy.